2025-03-25 01:15:17 error: Error connecting to database: syntax error at or near "NULL"
2025-03-25 01:15:39 error: Error connecting to database: syntax error at or near "NULL"
2025-03-25 01:23:56 error: Error connecting to database: syntax error at or near "NULL"
2025-03-25 01:24:10 error: Error connecting to database: syntax error at or near "NULL"
{"level":"error","message":"Unable to connect to the database: password authentication failed for user \"postgres\"","name":"SequelizeConnectionError","original":{"code":"28P01","file":"src\\backend\\libpq\\auth.c","length":115,"line":"307","name":"error","routine":"auth_failed","severity":"FATAL"},"parent":{"code":"28P01","file":"src\\backend\\libpq\\auth.c","length":115,"line":"307","name":"error","routine":"auth_failed","severity":"FATAL"},"service":"auth-service","stack":"SequelizeConnectionError: password authentication failed for user \"postgres\"\n    at Client._connectionCallback (C:\\Users\\aship\\Desktop\\Project-Kelmah\\kelmah-backend\\node_modules\\sequelize\\lib\\dialects\\postgres\\connection-manager.js:145:24)\n    at Client._handleErrorWhileConnecting (C:\\Users\\aship\\Desktop\\Project-Kelmah\\kelmah-backend\\node_modules\\pg\\lib\\client.js:336:19)\n    at Client._handleErrorMessage (C:\\Users\\aship\\Desktop\\Project-Kelmah\\kelmah-backend\\node_modules\\pg\\lib\\client.js:356:19)\n    at Connection.emit (node:events:519:28)\n    at C:\\Users\\aship\\Desktop\\Project-Kelmah\\kelmah-backend\\node_modules\\pg\\lib\\connection.js:116:12\n    at Parser.parse (C:\\Users\\aship\\Desktop\\Project-Kelmah\\kelmah-backend\\node_modules\\pg-protocol\\dist\\parser.js:36:17)\n    at Socket.<anonymous> (C:\\Users\\aship\\Desktop\\Project-Kelmah\\kelmah-backend\\node_modules\\pg-protocol\\dist\\index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)","timestamp":"2025-03-25 02:25:22"}
{"level":"error","message":"Unable to connect to the database: password authentication failed for user \"postgres\"","name":"SequelizeConnectionError","original":{"code":"28P01","file":"src\\backend\\libpq\\auth.c","length":115,"line":"307","name":"error","routine":"auth_failed","severity":"FATAL"},"parent":{"code":"28P01","file":"src\\backend\\libpq\\auth.c","length":115,"line":"307","name":"error","routine":"auth_failed","severity":"FATAL"},"service":"auth-service","stack":"SequelizeConnectionError: password authentication failed for user \"postgres\"\n    at Client._connectionCallback (C:\\Users\\aship\\Desktop\\Project-Kelmah\\kelmah-backend\\node_modules\\sequelize\\lib\\dialects\\postgres\\connection-manager.js:145:24)\n    at Client._handleErrorWhileConnecting (C:\\Users\\aship\\Desktop\\Project-Kelmah\\kelmah-backend\\node_modules\\pg\\lib\\client.js:336:19)\n    at Client._handleErrorMessage (C:\\Users\\aship\\Desktop\\Project-Kelmah\\kelmah-backend\\node_modules\\pg\\lib\\client.js:356:19)\n    at Connection.emit (node:events:519:28)\n    at C:\\Users\\aship\\Desktop\\Project-Kelmah\\kelmah-backend\\node_modules\\pg\\lib\\connection.js:116:12\n    at Parser.parse (C:\\Users\\aship\\Desktop\\Project-Kelmah\\kelmah-backend\\node_modules\\pg-protocol\\dist\\parser.js:36:17)\n    at Socket.<anonymous> (C:\\Users\\aship\\Desktop\\Project-Kelmah\\kelmah-backend\\node_modules\\pg-protocol\\dist\\index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)","timestamp":"2025-03-25 02:25:53"}
{"level":"error","message":"Redis connection error: connect ECONNREFUSED 127.0.0.1:6379","timestamp":"2025-03-26T02:09:57.879Z"}
{"level":"error","message":"Redis connection error: connect ECONNREFUSED 127.0.0.1:6379","timestamp":"2025-03-26T02:09:57.908Z"}
{"level":"error","message":"Redis connection error: connect ECONNREFUSED 127.0.0.1:6379","timestamp":"2025-03-26T02:09:57.972Z"}
{"level":"error","message":"Redis connection error: connect ECONNREFUSED 127.0.0.1:6379","timestamp":"2025-03-26T02:09:57.976Z"}
{"level":"error","message":"Redis connection error: connect ECONNREFUSED 127.0.0.1:6379","timestamp":"2025-03-26T02:09:58.076Z"}
{"level":"error","message":"Redis connection error: connect ECONNREFUSED 127.0.0.1:6379","timestamp":"2025-03-26T02:09:58.085Z"}
{"level":"error","message":"Redis connection error: connect ECONNREFUSED 127.0.0.1:6379","timestamp":"2025-03-26T02:09:58.237Z"}
{"level":"error","message":"Redis connection error: connect ECONNREFUSED 127.0.0.1:6379","timestamp":"2025-03-26T02:09:58.241Z"}
{"level":"error","message":"Failed to start authentication service: Cannot read properties of undefined (reading 'error')","stack":"TypeError: Cannot read properties of undefined (reading 'error')\n    at Sequelize.testConnection (C:\\Users\\aship\\Desktop\\Project-Kelmah\\kelmah-backend\\services\\auth-service\\config\\database.js:51:12)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async startServer (C:\\Users\\aship\\Desktop\\Project-Kelmah\\kelmah-backend\\services\\auth-service\\server.js:80:25)","timestamp":"2025-03-26T02:23:16.626Z"}
{"level":"error","message":"Unable to connect to the database: password authentication failed for user \"postgres\"","name":"SequelizeConnectionError","original":{"code":"28P01","file":"src\\backend\\libpq\\auth.c","length":115,"line":"307","name":"error","routine":"auth_failed","severity":"FATAL"},"parent":{"code":"28P01","file":"src\\backend\\libpq\\auth.c","length":115,"line":"307","name":"error","routine":"auth_failed","severity":"FATAL"},"stack":"SequelizeConnectionError: password authentication failed for user \"postgres\"\n    at Client._connectionCallback (C:\\Users\\aship\\Desktop\\Project-Kelmah\\kelmah-backend\\services\\auth-service\\node_modules\\sequelize\\lib\\dialects\\postgres\\connection-manager.js:145:24)\n    at Client._handleErrorWhileConnecting (C:\\Users\\aship\\Desktop\\Project-Kelmah\\kelmah-backend\\services\\auth-service\\node_modules\\pg\\lib\\client.js:336:19)\n    at Client._handleErrorMessage (C:\\Users\\aship\\Desktop\\Project-Kelmah\\kelmah-backend\\services\\auth-service\\node_modules\\pg\\lib\\client.js:356:19)\n    at Connection.emit (node:events:519:28)\n    at C:\\Users\\aship\\Desktop\\Project-Kelmah\\kelmah-backend\\services\\auth-service\\node_modules\\pg\\lib\\connection.js:116:12\n    at Parser.parse (C:\\Users\\aship\\Desktop\\Project-Kelmah\\kelmah-backend\\services\\auth-service\\node_modules\\pg-protocol\\dist\\parser.js:36:17)\n    at Socket.<anonymous> (C:\\Users\\aship\\Desktop\\Project-Kelmah\\kelmah-backend\\services\\auth-service\\node_modules\\pg-protocol\\dist\\index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)","timestamp":"2025-04-02T00:27:48.971Z"}
